biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -295,6 +308,15 @@ subroutine disturbance_rates( site_in, bc_in)
        currentPatch%disturbance_rates(dtype_ifire) = 0.0_r8
 
        dist_rate_ldist_notharvested = 0.0_r8
+
+       ! Avoid this calculation to avoid NaN due to division by zero result if luh is not used
+       if (hlm_use_luh .eq. itrue) then
+          currentPatch%landuse_transition_rates(1:n_landuse_cats) = min(1._r8, &
```

Comment:
@ckoven do you have a guide, or documentation on how the units of tracking these area changes flows from the datasets into the patch structures?  When dealing with fractions of fractions of physical area, things get hard to follow.